packages/imperative/src/cmd/src/CommandProcessor.ts

```diff
@@ -782,6 +782,10 @@ export class CommandProcessor {
         const secureInputs: Set<string> = new Set([...configSecureProps]);
         let censored = false;
 
+        const sessCfg: ISession = {};
+        ConnectionPropsForSessCfg.resolveSessCfgProps(sessCfg, commandParameters.arguments);
+        showInputsOnly.commandValues["authentication type"] = sessCfg.type;
```

Comment:
The `commandValues` section is for listing CLI arguments, so I don't know if it makes sense to add this there.

Could we add `authenticationType` as a new property at the top level (as sibling of `commandValues`)?